src/trivia/config.h.cmake

```diff
@@ -76,7 +76,7 @@
  */
 #cmakedefine ABORT_ON_LEAK 1
 /*
- * Defined if collecting leak backtrace have no issues.
+ * Defined if configured with ENABLE_BACKTRACE.
```

Comment:
I would keep the coment unchanged. We introduce `ENABLE_LEAK_BACKTRACE` to be able to disable the feature when something go wrong in some situations. So the comment reflects this. After the patch `ENABLE_LEAK_BACKTRACE` will be an alias for `ENABLE_BACKTRACE` but tomorrow it can be changed again.

On the other hand `ENABLE_LEAK_BACKTRACE` seems to be a workaround for the this particular bug. May be we should better disable `ENABLE_BACKTRACE` for aarch64 at that moment instead and not introduce `ENABLE_LEAK_BACKTRACE` at all. In this case we should just replace `ENABLE_LEAK_BACKTRACE` with `ENABLE_BACKTRACE` in the code. @locker what do you think?

Reply:
It doesn't look like we need `ENABLE_LEAK_BACKTRACE` now, when the bug is fixed. Let's replace it with `ENABLE_BACKTRACE`. I agree that if we find another issue with libunwind on aarch64, we'll be better off disabling `ENABLE_BACKTRACE` on aarch64 altogether.

Reply:
> I agree that if we find another issue with libunwind on aarch64, we'll be better off disabling ENABLE_BACKTRACE on aarch64 altogether.

Sounds arguable. I agree with @nshy we should retain this option for a while (and leave the comment untouched).

Reply:
> I agree that if we find another issue with libunwind on aarch64, we'll be better off disabling ENABLE_BACKTRACE on aarch64 altogether.

Sounds arguable. I agree with @nshy we should retain this option for a while (and leave the comment untouched).

Reply:
Removed `ENABLE_LEAK_BACKTRACE`.